src/OpenTelemetry.Contrib.Instrumentation.Runtime/OpenTelemetry.Contrib.Instrumentation.Runtime.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
+        <Description>dotnet runtime instrumentation for OpenTelemetry .NET</Description>
+        <PackageTags>$(PackageTags);OpenTelemetry;runtime</PackageTags>
+        <MinVerTagPrefix>Instrumentation.Runtime-</MinVerTagPrefix>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Extensions.Options" Version="2.1.1" />
```

Comment:
This dependency seems not required.

Reply:
Do we need to support `IDeferredMeterProviderBuilder`? This package is required (at the moment) to retrieve RuntimeMetricsOptions from the service provider.

Reply:
`IDeferredMeterProviderBuilder` is really nice to have, but not *required*. Some suggestions:
1. Maybe remove it for now and we can add it back later.
2. If keeping, consider checking the pattern followed in https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.StackExchangeRedis 

(My personal preference would be option1, so that we can quickly unblock this PR and do a 1st release.)

Reply:
Removed in order to unblock this PR.

Reply:
thanks. we can revisit this after an initial release.